docs/learning/solutions/cost-management/aws-list-unused-lambda.md

```diff
@@ -0,0 +1,104 @@
+# AWS - Identify Unused Lambda Functions
+
+## Description
+
+This automation job generates a listing of AWS Lambda functions and highlights any that may be eligible for deletion based on modification and execution dates provided as job inputs. It provides a detailed report of Lambda functions, including their last modified and last execution dates, and recommends whether to keep or delete each function.
+
+## Prerequisites
+
+- Turn on "[Runner as Node](/administration/runner/runner-management/node-dispatch.html#runner-as-a-node)" setting on your Runner.  
+  - This requires version 5.8.0 or higher.  Adjustments to Node tab may be required for earlier versions.
+- AWS CLI installed on the runner node
+- jq tool for JSON parsing installed on the runner node
+- Proper AWS credentials configured on the runner node
```

Comment:
add that it's recommended to install the Runner on an EC2, ECS container or in EKS so that the credentials can be inherited by associating an IAM role with the compute resource that the Runner is installed on.  Rather than setup CLI with static creds/keys

Reply:
and perhaps provide link to installing Runner: https://docs.rundeck.com/docs/administration/runner/runner-installation/runner-install.html

Reply:
We can add that as a tip box after merge